### DISCUSSION TITLE
Create a User with separate First name and Last Name

### DISCUSSION THREAD

**Discussion by jeverington** [statamic/cms#4384] [Troubleshooting] [answered]
Wondering how I can use the existing forms in the cp to create a user where I've specified a first_name and last_name instead of a name field?

**Comment by jasonvarga** [comment, marked as answer]
Edit the user blueprint, remove the `name` field, and add `first_name` and `last_name` fields.

**Reply by jeverington** [reply]
Thanks Jason. I've done that.
![image](https://user-images.githubusercontent.com/1991914/135930052-c25f28bf-c481-4cba-b91d-dda31887a12e.png)
Trying to create a user using the wizard requires a name to be set and doesn't ask for my custom fields. Because first name and last name are required it fails to create a user.
![image](https://user-images.githubusercontent.com/1991914/135930183-8c855267-281b-4eb9-a887-c74b12423257.png)
Also the column displays as blank for name in the list view
![image](https://user-images.githubusercontent.com/1991914/135930081-db1115e3-abef-42dd-91b5-03d2852163d0.png)

Wondered if there was a more elegant way than creating a user with no name set and then going back into the user once created to set their first and last name?

**Reply by ARCHES-Academy** [reply]
We're having the same issue, would love to be able to customize that user creation wizard, and the user listing page columns.

**Reply by jasonvarga** [reply]
Ah forgot all about the wizard. Yeah, these look like bugs.

**Reply by duncanmcclean** [reply]
Created a feature request for this so it's not lost - see statamic/ideas#670

**Reply by jasonvarga** [reply]
3.2.12 has been released with this fix. Replacing the `name` field with `first_name` and `last_name` fields in your blueprint should now show appropriately in the wizard and listings now.

**Reply by jeverington** [reply]
@jasonvarga can this also be sorted for search? Currently, the search still expects a name column when using a db.

**Reply by duncanmcclean** [reply]
@jeverington Open an issue and I'm sure that can be sorted!

**Reply by jeverington** [reply]
Issue opened [https://github.com/statamic/ideas/issues/818](https://github.com/statamic/ideas/issues/818)
